pagopa/io-app
# feat: [IOPLT-359] Upgrade `react-native-svg` to 15.1.0 and `@pagopa/io-app-design-system` to 1.30.0

## Short description
This PR upgrades the version of `react-native-svg` to major 15 version. The upgrade of DS library is mandatory to avoid possible conflicts between the both libraries.

Upgrade of cocoapods has been introduced to support the podspecs of svg library pointing to visionos.

## List of changes proposed in this pull request
- Upgrade `react-native-svg`
- Upgrade `@pagopa/io-app-design-system`
- Upgrade cocoapods

## How to test
Check the visualization of svgs on both Android and iOS.

Run following commands before running the app:
```
yarn install
gem install cocoapods@1.15.2
cd ios; pod install; cd ..
yarn run-ios
```
